libhb/dvd.c

```diff
@@ -242,6 +242,22 @@ static void add_subtitle( hb_list_t * list_subtitle, int position,
             break;
     }
 
+    switch (style)
+    {
+        case HB_VOBSUB_STYLE_4_3:
+            strcat( subtitle->lang, " (4:3)" );
```

Comment:
In my experience, brackets are commonly used for technical additions. Similar to in jurnalizm [sic]. No biggie, just avoids multiple sets of parens.